modules/macie-member-account/README.md

```diff
@@ -0,0 +1,46 @@
+# terraform-aws-tardigrade-macie/member
```

Comment:
The heading was changed, but the directory still needs to be renamed, from `macie-member-account` to just `member`.

Reply:
Directory renamed.